src/extension.ts

```diff
@@ -354,6 +363,47 @@ export function activate(context: vscode.ExtensionContext) {
     }));
 
     //
+    // Import of a GLB file and writing out its various chunks.
+    //
+    context.subscriptions.push(vscode.commands.registerCommand('gltf.importGlbOpen', () => {
+        const options: vscode.OpenDialogOptions = {
+             canSelectMany: false,
+             openLabel: 'Import',
+             filters: {
+                'Binary glTF': ['glb'],
+                'All files': ['*']
+            }
+        };
+
+        vscode.window.showOpenDialog(options).then(fileUri => {
+            if (fileUri && fileUri[0]) {
+                try {
+                    GlbImport.load(fileUri[0].fsPath);
+                } catch (ex) {
+                    vscode.window.showErrorMessage(ex.toString());
+                }
+            }
+        });
+    }));
+
+    context.subscriptions.push(vscode.commands.registerCommand('gltf.importGlbFile', (fileUri) => {
```

Comment:
After following my advice below about awaiting GlbImport.load, you'll have to declare this one as async.  I'm pretty sure this is the correct syntax, or at least it worked for me:

```
context.subscriptions.push(vscode.commands.registerCommand('gltf.importGlbFile', async (fileUri) => {
```